demo/demo.component.ts

```diff
@@ -125,6 +148,8 @@ import { Component, ViewEncapsulation } from '@angular/core';
   ]
 })
 export class DemoComponent {
+  // trick to force flatpickr loading even with build optimizers
+  localFlatpickr = flatpickr;
```

Comment:
Would all consumers of this lib be required to do this? I'd rather wait until https://github.com/flatpickr/flatpickr/pull/1310 gets merged so that we can just do:
```
import * as flatpickr from 'flatpickr';
```

and then use flatpickr directly for instantiating the instance. The current FLATPICKR token was the best workaround I could come up with in the meantime